sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1508,6 +1508,10 @@ def task_worker(self):
                         # D.2.2 Software Deinitialization
                         api.set_datapath_deinit(host_lanes_mask)
 
+                        self.log_notice("Applying App0 configuration for port {}".format(lport))
+                        if not api.apply_app0_configuration():
+                            self.log_notice("{}: Could not apply application 0 configurations".format(lport))
```

Comment:
Why this is a NOTICE and not Warning ?

Reply:
This is not a mandatory step, so I think it's preferable to use Notice